lib/screen/home/home_screen.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:event_bloc/event_bloc.dart';
+import 'package:flutter/material.dart';
+
+import 'package:simple_exercise_builder/screen/home/selected_workout.dart';
+import 'package:simple_exercise_builder/screen/home/start_workout.dart';
+
+class HomeScreen extends StatelessWidget {
+  final String workoutName = "Haron's Workout";
```

Comment:
You should prefer the const keyword for constants like this. Granted this is just test data, but it's good to keep in mind.

Also avoid unnecessary Type Indication. 
``` dart
const WORKOUT_NAME = "Haron's Workout";
```

Reply:
These mock data are in a Workout instance now so this is not needed anymore but I'll take note of unnecessary type indications.